### DISCUSSION TITLE
[Application] Filscout.com

### DISCUSSION THREAD

**Discussion by IPFSUnion** [filecoin-project/community#94] [Project Showcase]
## Project Description
[Filscout](https://filscout.com/) is a Filecoin Network browser developed by [IPFSUnion Team](https://www.ipfsunion.cn/), it's so convenient to get the Filecoin network chain data and do some data analytics. 

## Project Status

Filscout has released the first version, which has completed basic functions and can meet the most people's needs. More functions are beening developed, we will open an discussion in the GitHub, hope to get more useful suggestions.
## Previews

![image](https://user-images.githubusercontent.com/68763845/109381228-eea40c00-7913-11eb-83a8-a2047383b1a0.png)
![image](https://user-images.githubusercontent.com/68763845/109381238-11362500-7914-11eb-88c0-0abebe99eba5.png)
![image](https://user-images.githubusercontent.com/68763845/109381363-fe702000-7914-11eb-88c7-75f34b6612e1.png)
![image](https://user-images.githubusercontent.com/68763845/109381610-565b5680-7916-11eb-921b-e52901a1df38.png)

### Miner Gas Cost Analysis
[Filscout.com](https://filscout.com/)  has launched a new feature now **Miner Gas Cost Analysis**：This new feature will make it easy for users to view the **daily gas cost, power growth, lucky value**, and **block rewards**, so that the data on the chain is truly transparent!

In fact, it is a very complex process to count the gas used of each miner. It is necessary to collect each message and its gas used on the chain for performing summary statistics by day which is not easy. As a result, many users are unable to fully understand the data on the chain in a transparent and clear way, which has become a pain point for the industry.

This new feature by filscout.com will help all **customer client, business client** and **industry** clearly view the daily gas used and power growth of specified miner, so that users can see the FIL cost of each miner at a glance.

#### (Personal Computer)
![image](https://user-images.githubusercontent.com/68763845/112948849-213b6180-916b-11eb-8154-89ea2c70bd09.PNG)

#### (Mobile Device)
![image](https://user-images.githubusercontent.com/68763845/112951726-3bc30a00-916e-11eb-8cb5-07be4c325482.jpg)

### Statistics of orphan blocks
[Filscout.com](https://filscout.com/)  has released a new feature - Statistics of orphan blocks
: This feature allows you to view the number of orphan blocks in the whole block chain of Filecoin, including miners' address, tags, number of orphan blocks /percentage, sector size and ranking information.
	
Blocking optimization is an important part of the Filecoin mine pool solution, especially when it comes to effectively reducing the number of orphan blocks. As of 7:00 p.m. 2021-04-09 Beijing time, the number of awarded FIL in one block is 26.39FIL, one FIL is worth about $170, and one orphan blocks loses $4,486, equivalent to more than 290,00 RMB. Take a mine pool for example, the number of sorphan blocks in 7 days is 20, the loss of this pool is about 89,700 dollars, equivalent to about 590,000 yuan.

The single block statistics function launched by Filscout.com will help the miners timely find orphan blocks and timely analyze the possible reasons for the low lucky value.

![image](https://user-images.githubusercontent.com/68763845/114186879-f37bc700-9979-11eb-9641-49be763917bc.png)

![image](https://user-images.githubusercontent.com/68763845/114189276-9df4e980-997c-11eb-9380-773f97557d77.jpg)

## Target Audience

Everybody can use it to get the Filecoin Network chain infomations. Miners can get the relative mining infomation, clients can get the miners' status and chain  status and so on. 

## Comments
Filscout hopes to help Filecoin network give everybody a better experience. We are glad to collect more community advance to improve it. So if you have some suggestions or good ideas, please leave your comments or cantact us by email(tech@ipfsunion.cn).

**Comment by jennijuju** [comment]
Thank you for the updates and filscout is featured in[ Filecoin Newsletter #14](https://mailchi.mp/1407c088a2b3/filecoin-news-issue-66046)! I'm looking forward to more updates in the future!!

**Comment by jennijuju** [comment]
@IPFSUnion we would love to feature the latest updates of filscout in our showcase video. Is there any more updates you want to include?

**Comment by gdkiddd** [comment]
这里可以看到我们黑客松的filscout中文路演：
https://www.bilibili.com/video/BV1my4y1K76B?t=80
